internal/service/metricservice/dto.go

```diff
@@ -0,0 +1,40 @@
+package metricservice
+
+import (
+	"github.com/VadimOcLock/metrics-service/internal/store/somestore"
+)
+
+type UpdateGaugeDTO somestore.UpdateGaugeMetricParams
+
+func (dto UpdateGaugeDTO) Valid() error {
+	// may be later
```

Comment:
Рекомендую добавить TODO: к комментариям, показывающим намерение что-то зименить в будущем. Это позволит отделить их от обычных комментариев.

```suggestion
	// TODO: may be later
```